docs/whats-new/staging-release-notes.md

```diff
@@ -1,5 +1,22 @@
 # Staging release notes
 
+## October 2023
+
+**11 October 2023**
+
+Frontend version: 1.0.0-20231009.1523 | Backend version: 1.69.0-20231012.1057
+
+| Type      | Change      | Description |
+| --- | --- | --- |
+| **Feature** | Check existing users on Automation API | With the launch of CAM on 23 October 2023, the Automation API will now monitor terminated user events. 
```

Comment:
I suggest:
With the launch of CAM on 23 October 2023, the new [Get Existing Users API](https://stg.docs.developer.tech.gov.sg/docs/techpass-automation-api/#tag/User/paths/~1iam~1existing-users/post) facilitates a one-off event where tenants can clean up their local user store by removing accounts that no longer exist in TechPass AAD. This step is vital before integration with TechPass for CAM compliance. After integration, tenants will receive new user terminated webhook events.